js/stable.js

```diff
@@ -724,9 +719,9 @@ dxSTable.prototype.colDragMoveEnd = function(e) {
 dxSTable.prototype.scrollPos = function()
 {
 	this.scp.style.display = "block";
-	var mni = Math.floor(this.dBody.scrollTop / TR_HEIGHT);
-	var mxi = mni + Math.floor(this.dBody.clientHeight / TR_HEIGHT);
-	var mid = Math.floor(((mni + mxi) / 2));
+	var mni = Math.floor(this.dBody.scrollTop / this.TR_HEIGHT);
+	var mxi = mni + Math.floor(this.tBody.height() / this.TR_HEIGHT);
```

Comment:
While we're at it, why not expand the variable name to make it easier to read? This should be `maximum` or `maxi`?